apps/protocol-frontend/src/components/AttestationsTableShell.tsx

```diff
@@ -33,7 +36,7 @@ const AttestationsTableShell = () => {
       guilds: {
         some: {
           guild_id: {
-            in: userData?.guild_users.map(guild => guild.guild_id) || [],
+            in: guildIds,
```

Comment:
I like this, it's a much cleaner and readable way to have this.  I see these suggestions in your PR review that we should pull out this type of logic into variables so that it's more readable. We should include this in our best practices at this point 🙌